<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow </class>
 <widget class="QMainWindow" name="MainWindow ">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Домино</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string>background-color: #b4c28a;</string>
   </property>
   <widget class="QPushButton" name="playButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>227</x>
      <y>300</y>
      <width>345</width>
      <height>65</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>778</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: #eaf2e0;
 border-radius: 12px;
 font-size: 20px;
 color: #222;
 border: 2px solid #666;
 min-width: 60px;
 min-height: 60px;
}
           </string>
    </property>
    <property name="text">
     <string>Играть</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="rulesButton">
    <property name="geometry">
     <rect>
      <x>227</x>
      <y>395</y>
      <width>345</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: #eaf2e0;
 border-radius: 12px;
 font-size: 20px;
 color: #222;
 border: 2px solid #666;
 min-width: 60px;
 min-height: 60px;
}
           </string>
    </property>
    <property name="text">
     <string>Правила игры</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>5</y>
      <width>654</width>
      <height>327</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="icon.qrc">:/icon/domino.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="settingsButton">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>507</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: #eaf2e0;
 border-radius: 12px;
 font-size: 20px;
 color: #222;
 border: 2px solid #666;
 min-width: 60px;
 min-height: 60px;
}
           </string>
    </property>
    <property name="text">
     <string>Настройки</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/icon/setting.png</normaloff>:/icon/setting.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="statsButton">
    <property name="geometry">
     <rect>
      <x>564</x>
      <y>507</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: #eaf2e0;
 border-radius: 12px;
 font-size: 20px;
 color: #222;
 border: 2px solid #666;
 min-width: 60px;
 min-height: 60px;
}
           </string>
    </property>
    <property name="text">
     <string>Статистика</string>
    </property>
    <property name="icon">
     <iconset resource="icon.qrc">
      <normaloff>:/icon/stat.png</normaloff>:/icon/stat.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <zorder>settingsButton</zorder>
   <zorder>statsButton</zorder>
   <zorder>label</zorder>
   <zorder>playButton</zorder>
   <zorder>rulesButton</zorder>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
